minorminer/utils/embeddings.py

```diff
@@ -131,18 +136,32 @@ def visualize_embeddings(
     )
 
 
-def shuffle_graph(T, prng=None):
-    """Shuffle the node and edge ordering of a networkx graph."""
+def shuffle_graph(G: nx.Graph, prng=None) -> nx.Graph:
+    """Shuffle the node and edge ordering of a networkx graph.
+
+    For embedding methods that operate as a function of the node or edge
+    ordering this can force diversification in the returned embeddings. Note
+    that special orderings that encode graph structure or geometry may enhance
+    embedder performance (shuffling may lead to longer search times).
+
+    Args:
+        G (nx.Graph): A networkx graph
+        prng (np.random.Generator, optional): When provided, is used to shuffle
+            the order of nodes and edges in the source and target graph. This
+            can allow sampling from otherwise deterministic routines.
+    Returns:
+        nx.Graph: The same graph with modified node and edge ordering.
+    """
     if prng is None:
         prng = np.random.default_rng()
-    nodes = list(T.nodes())
+    nodes = list(G.nodes())
     prng.shuffle(nodes)
-    edges = list(T.edges())
+    edges = list(G.edges())
     prng.shuffle(edges)
-    _T = nx.Graph()
-    _T.add_nodes_from(nodes)
-    _T.add_edges_from(edges)
-    return _T
+    _G = nx.Graph()
+    _G.add_nodes_from(nodes)
+    _G.add_edges_from(edges)
+    return _G
```

Comment:
```suggestion
    H = nx.Graph()
    H.add_nodes_from(nodes)
    H.add_edges_from(edges)
    return H
```
or even `G`

Reply:
I think H was usued in the original visualize subgraph function inside Ocean.